com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -1242,10 +1258,10 @@ private bool ExportInstance(GameObject unityGo, FbxNode fbxNode)
 
             FbxMesh fbxMesh = null;
 
-            if (!SharedMeshes.TryGetValue (unityPrefabParent.name, out fbxMesh))
+            if (!SharedMeshes.TryGetValue (unityPrefabParent.GetInstanceID(), out fbxMesh))
```

Comment:
Is GetInstanceID consistent if a user closes and restarts Unity? If it is, then what happens if the Library folder is deleted. Does it still map?

Reply:
I might be misunderstanding but it only needs to be consistent for the duration of the export no? If the user closes and reopens Unity a new ID will be generated, however the map won't exist anymore.

Reply:
It was my misunderstanding, I thought we exported the Unity ID.